Tasks/Common/ServiceFabricHelpers/Connect-ServiceFabricClusterFromServiceEndpoint.ps1

```diff
@@ -158,6 +158,14 @@ function Connect-ServiceFabricClusterFromServiceEndpoint {
                 $clusterConnectionParameters["X509Credential"] = $true
             }
         }
+        else
+        {
+            if ($ConnectedServiceEndpoint.Auth.Parameters.IsWindowsCredential -eq "true")
```

Comment:
Do we know what happens if specifying this parameter against a non-secure cluster? I've reached out to Service Fabric to get a reply. If it will work, maybe we could just always do that.